File: app/src/main/java/com/erebor/tomkins/pos/data/converters/Generators.java
package com.erebor.tomkins.pos.data.converters;

import java.util.UUID;

public class Generators {
    public static String getRandomUUID() {
        UUID uuid = UUID.randomUUID();
        String uuidString = String.valueOf(uuid);

        return uuidString.replaceAll("-", "");
    }
}

File: app/src/main/java/com/erebor/tomkins/pos/data/converters/Generators.kt
package com.erebor.tomkins.pos.data.converters

import java.util.UUID

object Generators {
    val randomUUID: String
        get() {
            val uuid = UUID.randomUUID()
            val uuidString = uuid.toString()

            return uuidString.replace("-".toRegex(), "")
        }
}
